forte/data/readers/largemovie_reader.py

```diff
@@ -0,0 +1,116 @@
+# Copyright 2019 The Forte Authors. All Rights Reserved.
```

Comment:
I just realize we should change the copyright to 2020 now

Reply:
> I just realize we should change the copyright to 2020 now

@hunterhector Should I apply the change of 2020 to all the other files in this project?

Reply:
Let's just use 2020 for new files. I'm currently not sure what I should do about the rest, yet.